samples/word/35-ranges/get-pages.yaml

```diff
@@ -0,0 +1,244 @@
+order: 4
+id: word-ranges-get-pages
+name: 'Work with pages, panes, and windows'
+description: 'Shows how to work with pages, panes, and windows APIs.'
+author: yilin4
+host: WORD
+api_set:
+    WordApiDesktop: '1.2'
+script:
+    content: |
+        $("#get-pages-selected-range").on("click", () => tryCatch(getPagesOfSelectedRange));
+        $("#get-pages-third-paragraph").on("click", () => tryCatch(getPagesOfThirdParagraph));
+        $("#get-pages-enclosing-viewport").on("click", () => tryCatch(getPagesEnclosingViewport));
+        $("#get-all-pages").on("click", () => tryCatch(getAllPages));
+        $("#setup").on("click", () => tryCatch(setup));
+
+        async function getPagesOfSelectedRange() {
+          await Word.run(async (context) => {
+            // Gets pages of the selection.
+            const pages: Word.PageCollection = context.document.getSelection().pages;
+            pages.load();
+            await context.sync();
+
+            console.log(pages);
+            // Log info for pages included in selection.
+            for (let i = 0; i < pages.items.length; i++) {
+              let page = pages.items[i];
+              page.load();
+              await context.sync();
```

Comment:
We should avoid doing load/sync in a loop. It's a bad pattern. This article has some tips to avoid that: https://learn.microsoft.com/en-us/office/dev/add-ins/concepts/correlated-objects-pattern